pkg/kubelet/container/helpers.go

```diff
@@ -126,6 +127,17 @@ func pickFieldsToHash(container *v1.Container) map[string]string {
 	return retval
 }
 
+// HashAuth - returns a hash code for a CRI pull image auth, and an error if the hash cannot be calculated.
```

Comment:
A small nit: Go style comments are slightly different.

Reply:
It is explaining what it does and returns. What more can we add?

Reply:
It's not about what it does, but how it does it. Look at some other Go function comments. There is a specific comment syle Go follows.